charts/karma/Chart.yaml

```
@@ -6,7 +6,7 @@ home: https://github.com/prymitive/karma
 sources:
   - https://github.com/wiremind/wiremind-helm-charts/tree/main/charts/karma
   - https://github.com/prymitive/karma
-version: 2.1.4
+version: 2.1.5
```

```suggestion
version: 2.2.0
```
Since you add a new feature!